SCClassLibrary/Common/Control/Server.sc

```diff
@@ -38,8 +40,6 @@ ServerOptions {
 	var <>threads = nil; // for supernova
 	var <>useSystemClock = false;  // for supernova
 
-	var <numPrivateAudioBusChannels=112;
```

Comment:
Comment at start of file says "order of variables is important here. Only add new instance variables to the end"... is this something to worry about?